content/tutorials/geodesic-getting-started.md

```diff
@@ -0,0 +1,97 @@
+
+In this tutorial, we'll walk you through how to use Geodesic to execute an authenticated AWS CLI command and talk about what is going on under the hood.
+
+## Prerequisites
+
+### Requirements
+
+To accomplish this tutorial, you'll need to have an AWS Account handy and have [Docker installed](https://docs.docker.com/get-docker/) on your local machine. **That's all**.
+
+### Docker Primer
+
+Before we jump in, it's important to note that Geodesic is built around some advanced features of Docker and the Docker CLI that are worth understanding. If you know the docker CLI well then feel free to skip over this section.
+
+1. Geodesic overrides the default entrypoint with a custom script that enables starting a login shell or installing geodesic as an executable on your local machine. If you don't know much about Docker's `ENTRYPOINT` or `CMD` capabilities then [we recommend reading this article before moving forward](https://phoenixnap.com/kb/docker-cmd-vs-entrypoint).
+1. Geodesic can be thought of as just another shell that you open up on your machine like `zsh` or `bash`, but when running it inside of Docker means that it doesn't have access to your home directory or the projects you're looking to work on. To get around that, Geodesic is typically run using Docker volume bind mounts via the `--volume` / `-v` flag. You'll see this in our examples below, but if you would like to understand more about what is going on under the hood then we recommend [reading up quickly on mounting local volumes via `docker run`](https://docs.docker.com/engine/reference/commandline/run/#mount-volume--v---read-only).
+
+### Geodesic Usage Patterns
+
+Let's talk about a few of the ways that one can run Geodesic. Our toolbox has been built to satisfy many use-cases, and each result in a different pattern of invocation:
+
+1. You can **run standalone** Geodesic as a standard docker container using `docker run`. This enables you to get started quickly, avoid fiddling with configuration or run one-off commands using some of the built-in tools.
+   1. Example: `docker run -it --rm --volume $HOME:/localhost cloudposse/geodesic:latest-debian --login` opens a bash login shell (`--login` is our Docker `CMD` here) in our Geodesic container.
+   1. Example: `docker run -it --rm --volume $HOME:/localhost cloudposse/geodesic:latest-debian -c "terraform version"` executes the `terraform version` command as a one off and outputs the result.
+1. You can **install** Geodesic onto your local machine using what we call the docker-pipe-bash pattern (e.g. `docker run ... | bash`). Similar to above, this enables a quickstart process but supports longer lived usage as it creates a callable script on your machine that enables reuse any time you want to start a shell.
```

Comment:
```suggestion
     1. You can **install** Geodesic onto your local machine using what we call the docker-pipe-bash pattern (e.g. `docker run ... | bash`). Similar to above, this enables a quickstart process but supports longer lived usage as it creates a callable script on your machine that enables reuse any time you want to start a shell.
```

Reply:
@osterman Did you want to indent this under the initial usage pattern to combine them? Or was this a mistake?

Reply:
No strong preference, I just saw this:
![image](https://user-images.githubusercontent.com/52489/110022962-6b066880-7ce1-11eb-999f-cdd1039f533d.png)

Let's just make it consistent one way or the other.

Reply:
Ok, now I understand. Ignore. I retardedly didn't realize you were indenting an ordered list because when everything is just 1., 1., 1. it is indistinguishable from a typo. =)